LocationFetcher updated to support API 23 (M).

teamagam1 opened the pull request:
LocationFetcher updated to support API 23 (M).

The registration to the location fetcher was moved to the Launcher Activity.

teamagam1 committed:
LocationFetcher updated to support API 23 (M).

The register to the location fetcher was moved to the Launcher Activity.

review-ninja commented:
<a href="https://app.review.ninja/teamagam/GimelGimel/pull/15" target="_blank"><img src="https://app.review.ninja/52010471/pull/15/badge" alt="ReviewNinja"/>

teamagam1 committed:
LocationFetcher updated to support Broadcasting Location updates by LocationManager.

request for location permission is done in the launcher activity.

register is in viewerFragmant and Launcher.

gRaytan commented on app/src/main/java/com/teamagam/gimelgimel/app/GGApplication.java:
```diff
@@ -39,41 +30,17 @@ public void onCreate() {
 
         CheckIfAppUpdated();
```
mPrefSecureKey
what is it?
should be deleted

gRaytan replied:
CheckIfAppUpdated - all in comments, why?

gRaytan commented on app/src/main/java/com/teamagam/gimelgimel/app/GGApplication.java:
```diff
@@ -39,41 +30,17 @@ public void onCreate() {
-            }
-        });
-        mLocationFetcher.addProvider(LocationFetcher.ProviderType.LOCATION_PROVIDER_GPS);
-//        mLocationFetcher.addProvider(LocationFetcher.ProviderType.LOCATION_PROVIDER_NETWORK);
```
in comment, we need it or not?

gRaytan replied:
never mind

gRaytan commented on app/src/main/java/com/teamagam/gimelgimel/app/GGApplication.java:
```diff
@@ -39,41 +30,17 @@ public void onCreate() {
     @Override
     public void onTerminate() {
         super.onTerminate();
-        mLocationFetcher.unregisterFromUpdates();
```
you sure tahts the right order?

gRaytan replied:
never mind

gRaytan commented on app/src/main/java/com/teamagam/gimelgimel/app/control/sensors/LocationFetcher.java:
```diff
@@ -23,8 +27,6 @@
  */
 public class LocationFetcher {
 
-    private static final String LOG_TAG = LocationFetcher.class.getSimpleName();
```
y? not in use? how will we write logs?
